### ISSUE TITLE
Upload button missing label

### ISSUE THREAD

**Issue by jacobbp25** [microsoft/BotFramework-WebChat#5230]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

What It Means
A form control does not have a corresponding label.
Why It Matters
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.

### What browsers and screen readers do this issue affect?

Others

### Are there any code-based customization done to Web Chat?

Yes, however, this also repro when customizations are not in use.

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Inspect web chat client with hideUploadButton: false using WAVE tool

Tool reports missing label from <input aria-disabled="false" aria-hidden="true" class="webchat__upload-button--file-input" multiple="" role="button" tabindex="-1" type="file">

### What do you expect?

Input to have label to meet standards

### What actually happened?

Input missing label

### Did you find any DOM elements that might have caused the issue?

<input aria-disabled="false" aria-hidden="true" class="webchat__upload-button--file-input" multiple="" role="button" tabindex="-1" type="file">
